WooCommerce/Classes/ViewRelated/ReusableViews/TwoColumnHeadlineFootnoteTableViewCell.swift

```diff
@@ -32,15 +34,16 @@ final class TwoColumnHeadlineFootnoteTableViewCell: UITableViewCell {
         }
     }
 
-    /// Footnote label text
-    ///
-    var footnoteText: String? {
-        get {
-            return footnoteLabel?.text
-        }
-        set {
-            footnoteLabel?.text = newValue
-        }
+    func updateFootnoteAttributedText(_ attributedString: NSAttributedString?) {
+        footnoteLabel.attributedText = attributedString
+    }
+
+    func updateFootnoteText(_ footnoteText: String?) {
+        footnoteLabel.text = footnoteText
+    }
+
+    func toggleFootnote() {
+        footnoteLabel.isHidden = !footnoteLabel.isHidden
```

Comment:
hm I was wondering when this would be the use case? since this cell class `TwoColumnHeadlineFootnoteTableViewCell` could be reused for different index paths, the `footnoteLabel` could be shown/hidden by other index paths before

Reply:
changed to `hideFootnote()`